recipe/meta.yaml

```diff
@@ -48,9 +48,9 @@ test:
     - gmt defaults -Vd  # [not linux]
     - gmt psbasemap -R10/70/-3/8 -JX4i/3i -Ba -B+glightred+t"TEST" -P -Vd > test0.ps  # [not linux]
     - gmt pscoast -R0/5/0/5 -JM5i -P -W0.25p -Vd > test1.ps  # [not linux]
-    - gmt begin -Vd  # [osx]
-    - gmt coast -R0/5/0/5 -JM5i -W0.25p -Vd  # [osx]
-    - gmt end  # [osx]
+    - gmt begin -Vd  # [not linux]
+    - gmt coast -R0/5/0/5 -JM5i -W0.25p -Vd  # [not linux]
+    - gmt end  # [not linux]
```

Comment:
Why these tests are not working on Linux?

Reply:
It was something to do with the huge paths used by conda causing segfaults. Not sure if they are still happening actually.